src/ansys/fluent/core/session.py

```diff
@@ -126,6 +127,30 @@ def exit(self) -> None:
         """Close the Fluent connection and exit Fluent."""
         self.fluent_connection.exit()
 
```

Comment:
@prmukherj scheme_eval returns a value. Ideally, start_journal should return a value if we can do that. Then that value can be passed to stop_journal, right?

cc: @mkundu1